src/components/Location/LocationSearch.tsx

```diff
@@ -20,7 +20,7 @@ import locationApi from "@/types/location/locationApi";
 
 interface LocationSearchProps {
   facilityId: string;
-  mode?: "kind" | "location";
+  mode?: "kind" | "location" | "instance";
```

Comment:
Location is not a valid mode
```suggestion
  mode?: "kind" | "instance";
```